pages/newsletter.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <div>
+    <paragraphComponent v-for="(paragraph, index) in paragraphs" :key="index" type="type" classes="classes" content="content" generated_html="generated_html" index="index"/>
+  </div>
+</template>
+
+<script>
+import paragraphComponent from '~/components/newfoldercomponent/ParagraphComponent.vue';
+
+
+export default {
+    components:{
+        paragraphComponent
+    },
+    data(){
+    return {
+          types :{
+
+            normal_text: {
+              'bold': true,
+              'italic': true,
+              'underline': true,
+
+            },
+            main_heading: {
+              'bold': false,
+              'italic': true,
+              'underline': true,
+
+            },
+            sub_heading: {
+              'bold': false,
+              'italic': true,
+              'underline': true,
+
+            },
+            quote: {
+              'bold': false,
+              'italic': true,
+              'underline': true,
+
+            },
+
+          },
+        actions:{
+            bold: 'flok-nl-bold',
+            underline: 'flok-nl-underline',
+            italics: 'flok-nl-italics',
+            quote: 'flok-nl-quote',
+            main_heading: 'flok-nl-main-heading',
+            sub_heading: 'flok-nl-sub-heading',
+
+        },
+
+         paragraphs: [   
+                    {
+                        type: 'normal-text',
+                        content: 'Hello world',
+                        classes: ['flok-nl-bold', 'flok-nl-underline'],
+                        generated_html: '<div class="flok-nl-bold flok-nl-underline">Hello World</div>'
+                    },
+                    {
+                        type: 'normal-text',
+                        content: 'Hello world',
+                        classes: ['flok-nl-italic', 'flok-nl-underline'],
+                        generated_html: '<div class="flok-nl-italic flok-nl-underline">Hello World</div>'
+                    },
+                    {
+                        type: 'quote',
+                        content: 'Hello world',
+                        classes: ['flok-nl-quote'],
+                        generated_html: '<div class="flok-nl-quote">Hello World</div>'
+                    },
```

Comment:
You can delete this. According to the document on page initialization there should be only 1 object in this array